docs/dc-apps-performance-toolkit-user-guide-bitbucket.md

```diff
@@ -23,7 +32,211 @@ For simple spikes or tests, you can skip steps 1-2 and target any Bitbucket test
+| Database instance class | [db.t3.medium](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html#Concepts.DBInstanceClass.Summary) |
+| RDS Provisioned IOPS | 1000 |
+| Master password | Password1! |
+| Enable RDS Multi-AZ deployment | false |
+| Bitbucket database password | Password1! |
+| Database storage | 100 |
+
+**Elasticsearch**
+
+| Parameter | Recommended Value |
+| --------- | ----------------- |
+| Elasticsearch master user password | (leave blank) |
+| Elasticsearch instance type | m4.xlarge.elasticsearch |
+| Elasticsearch disk-space per node (GB) | 1000 |
+
+**Networking (for new ASI)**
+
+| Parameter | Recommended Value |
+| --------- | ----------------- |
+| Trusted IP range | 0.0.0.0/0 _(for public access) or your own trusted IP range_ |
+| Availability Zones | _Select two availability zones in your region_ |
+| Permitted IP range | 0.0.0.0/0 _(for public access) or your own trusted IP range_ |
+| Make instance internet facing | true |
+| Key Name | _The EC2 Key Pair to allow SSH access. See [Amazon EC2 Key Pairs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) for more info._ |
+
+**Networking (for existing ASI)**
+
+| Parameter | Recommended Value |
+| --------- | ----------------- |
+| Make instance internet facing | true |
+| Permitted IP range | 0.0.0.0/0 _(for public access) or your own trusted IP range_ |
+| Key Name | _The EC2 Key Pair to allow SSH access. See [Amazon EC2 Key Pairs](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html) for more info._ |
+
+### Running the setup wizard
+
+After successfully deploying Bitbucket Data Center in AWS, you'll need to configure it:
+
+1. In the AWS console, go to **Services** > **CloudFormation** > **Stack** > **Stack details** > **Select your stack**.
+1. On the **Outputs** tab, copy the value of the **LoadBalancerURL** key.
+1. Open **LoadBalancerURL** in your browser. This will take you to the Bitbucket setup wizard.
+1. On the **Bitbucket setup** page, populate the following fields:
+    - **Application title**: any name for your Bitbucket Data Center deployment
+    - **Base URL**: your stack's Elastic LoadBalancer URL
+    - **License key**: select new evaluation license or existing license checkbox
+    Click **Next**.
+1. On the **Administrator account setup** page, populate the following fields:
+    - **Username**: admin _(recommended)_
+    - **Full name**: any full name of the admin user
+    - **Email address**: email address of the admin user
+    - **Password**: admin _(recommended)_
+    - **Confirm Password**: admin _(recommended)_
+    Click **Go to Bitbucket**.
+
+---
+
+### <a id="devdataset"></a>2. Generate dataset for development environment  
+
+After creating the development environment Bitbucket Data Center, generate test dataset to run Data Center App Performance Toolkit:
+- Create project
+- Create repository in project
+- Create pull request 
```

Comment:
Create a new branch from the repo, make and push changes to the branch and create a pull request